packages/account/src/Sections/Profile/PhoneVerification/cancel-phone-verification-modal.tsx

```diff
@@ -1,57 +1,59 @@
     should_show_cancel_verification_modal: boolean;
     setShouldShowCancelVerificationModal: (value: boolean) => void;
 };
 
-const CancelPhoneVerificationModal = ({
-    should_show_cancel_verification_modal,
-    setShouldShowCancelVerificationModal,
-}: TCancelPhoneVerificationModal) => {
-    const history = useHistory();
-    const handleCancelButton = () => {
-        setShouldShowCancelVerificationModal(false);
-        history.goBack();
-    };
+const CancelPhoneVerificationModal = observer(
+    ({
+        should_show_cancel_verification_modal,
+        setShouldShowCancelVerificationModal,
+    }: TCancelPhoneVerificationModal) => {
+        const history = useHistory();
+        const handleCancelButton = () => {
+            setShouldShowCancelVerificationModal(false);
+            history.goBack();
+        };
+        const { ui } = useStore();
+        const { is_mobile } = ui;
 
-    return (
-        <Modal className='phone-verification__cancel-modal' is_open={should_show_cancel_verification_modal}>
-            <Modal.Body>
-                <div className='phone-verification__cancel-modal--contents'>
-                    <Text bold>
-                        <Localize i18n_default_text='Cancel phone number verification?' />
-                    </Text>
-                    <Text>
-                        <Localize i18n_default_text='All details entered will be lost.' />
-                    </Text>
-                </div>
-            </Modal.Body>
-            <Modal.Footer>
-                <div className='phone-verification__cancel-modal--buttons'>
-                    <Button
-                        color='black'
-                        fullWidth
-                        size='lg'
-                        onClick={() => setShouldShowCancelVerificationModal(false)}
-                    >
-                        <Text color='white' bold>
-                            <Localize i18n_default_text='Go back' />
-                        </Text>
-                    </Button>
-                    <Button variant='secondary' color='black' fullWidth size='lg' onClick={handleCancelButton}>
+        return (
+            <Modal
+                isMobile={is_mobile}
+                showHandleBar
+                isOpened={should_show_cancel_verification_modal}
+                primaryButtonCallback={() => setShouldShowCancelVerificationModal(false)}
+                primaryButtonLabel={<Localize i18n_default_text='Go back' />}
+                disableCloseOnOverlay
+                showSecondaryButton
+                secondaryButtonLabel={<Localize i18n_default_text='Yes, cancel' />}
+                secondaryButtonCallback={handleCancelButton}
+            >
+                <Modal.Header
+                    image={<LabelPairedCircleXmarkLgRegularIcon fill='#C40000' height={96} width={96} />}
+                    style={{
+                        backgroundColor: 'var(--core-color-solid-red-100)',
```

Comment:
Hi @suisin-deriv 
is this `--core-color-solid-green-100` coming from quill token ?
if it's from quill-tokens. Then, we should have class for it and I guess we should use class instead of inline css. 

IMHO even we don't have any class from quill then maybe we can create our own class instead of inline css 😬 

Thank you